content/docs/ref/pr.md

```diff
@@ -0,0 +1,20 @@
+# Command Reference: `pr`
+
+Commit specified files to a new branch and create a pull request.
+
+```dvc
+cml pr '**/*.py' '**/*.json'
+```
+
+is roughly equivalent to:
+
+```dvc
+SHA="$(git log -n1 --format=%h)"
+BASE="$(git branch)"
+git checkout -b "${BASE}-cml-pr-${SHA}"
+git add *.py *.json
+git commit -m "CML PR for ${SHA} [skip ci]"
+git push
+gh pr create -B ${BASE}
+git checkout -
```

Comment:
```suggestion
```

Just re-looked at the implementation and this seems to be removed now?

Reply:
Yes, it doesn't work like that anymore (if it ever did), and it [might not be what users expect](https://discord.com/channels/485586884165107732/728693131557732403/888777744220049478)

Reply:
@casperdcl, it's probably material for a separate issue, but the short answer is that it currently remains on the newly created branch. 🙃